rust/libits-client/src/mqtt/topic/message_type.rs

```diff
@@ -54,6 +58,8 @@ impl convert::From<&str> for MessageType {
             "denm" => MessageType::DENM,
```

Comment:
you may replace `impl cmp::PartialEq` with `impl PartialEq` and remove the import.

and you may remove all the `convert::`